test/sql/updates/setup.repair.sql

```diff
@@ -7,175 +7,188 @@
+     SELECT dimension_slice_id FROM slices
+     WHERE hypertable = 'repair_test_extra'::regclass
+     );
+
+     -- Break the partition constraints on some of the tables. The
+     -- partition constraints look the same in all tables so we create a
+     -- mix of tables with no missing partition constraint slices, just one
+     -- missing partition constraint dimension slice, and several missing
+     -- partition constraint dimension slices.
+     DELETE FROM _timescaledb_catalog.dimension_slice WHERE id IN (
+     SELECT dimension_slice_id FROM slices
+     WHERE hypertable = 'repair_test_timestamp'::regclass AND column_name = 'tag'
+     ORDER BY dimension_slice_id LIMIT 1
+     );
+
+     DELETE FROM _timescaledb_catalog.dimension_slice WHERE id IN (
+     SELECT dimension_slice_id FROM slices
+     WHERE hypertable = 'repair_test_date'::regclass AND column_name = 'tag'
+     ORDER BY dimension_slice_id
+     );
+
+     \echo **** Expected repairs ****
+     WITH unparsed_slices AS (
+     SELECT dimension_id,
+          dimension_slice_id,
+          hypertable,
+          constraint_name,
+          column_type,
+          column_name,
+          (SELECT SUBSTRING(constraint_expr, $$>=\s*'?([\w\d\s:+-]+)'?$$)) AS range_start,
+          (SELECT SUBSTRING(constraint_expr, $$<\s*'?([\w\d\s:+-]+)'?$$)) AS range_end
+          FROM slices
+     )
+     SELECT DISTINCT
+          dimension_slice_id,
+          dimension_id,
+          CASE
+          WHEN column_type = 'timestamptz'::regtype THEN
+               EXTRACT(EPOCH FROM range_start::timestamptz)::bigint * 1000000
+          WHEN column_type = 'timestamp'::regtype THEN
+               EXTRACT(EPOCH FROM range_start::timestamp)::bigint * 1000000
+          WHEN column_type = 'date'::regtype THEN
+               EXTRACT(EPOCH FROM range_start::date)::bigint * 1000000
+          ELSE
+               CASE
+               WHEN range_start IS NULL
+               THEN (-9223372036854775808)::bigint
+               ELSE range_start::bigint
+               END
+          END AS range_start,
+          CASE
+          WHEN column_type = 'timestamptz'::regtype THEN
+               EXTRACT(EPOCH FROM range_end::timestamptz)::bigint * 1000000
+          WHEN column_type = 'timestamp'::regtype THEN
+               EXTRACT(EPOCH FROM range_end::timestamp)::bigint * 1000000
+          WHEN column_type = 'date'::regtype THEN
+               EXTRACT(EPOCH FROM range_end::date)::bigint * 1000000
+          ELSE
+               CASE WHEN range_end IS NULL
+               THEN 9223372036854775807::bigint
+               ELSE range_end::bigint
+               END
+          END AS range_end
+     FROM unparsed_slices
+     WHERE dimension_slice_id NOT IN (SELECT id FROM _timescaledb_catalog.dimension_slice);
+
+     DROP VIEW slices;
+\endif
+
+\if :has_cagg_joins
+     \ir setup.repair.cagg_joins.sql
+\endif
 \ir setup.repair.cagg.sql
```

Comment:
```suggestion
\ir setup.repair.cagg.sql
\if :has_cagg_joins
     \ir setup.repair.cagg_joins.sql
\endif
```
We're adding a new repair test so make sense to be added at the end.